popgen/GvcfZipper.go

```diff
@@ -0,0 +1,80 @@
+package popgen
+
+import (
+	"sort"
+	"strings"
+	"log"
+	"github.com/vertgenlab/gonomics/vcf"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+)
+
+//TajimaZipperVcf returns the Tajima's D value for each entry in a sorted bed slice using a pre-SORTED gVCF file for variant information.
+func TajimaZipperVcf(b []*bed.Bed, gVCFFile string) []float64 {
+	var answer []float64
+	var all []*IndividualAllele
+	answer = make([]float64, 0)
+	alpha := vcf.GoReadGVcf(gVCFFile)//must be sorted beforehand
+	sort.Sort(bed.ByGenomicCoordinates{b})
+	var currBedIndex int = 0
+	var currVariants []*vcf.Vcf
+	currVariants = make([]*vcf.Vcf, 0)
+	for i := range alpha.Vcfs {
+		for 1 > 0 {//infinite condition for break loop
```

Comment:
This is fine, but just in case you didn't know you can also make an infinite loop by not giving any arguments to for. e.g. 
`for {
...
if x {break}
}`